src/schml/insert-casts.rkt

```diff
@@ -267,9 +280,64 @@
               ;; Justification: the runtime type is at
               ;; least as precise as t by semantics.
               (MBoxCastedSet! e1 e2 e2-ty))]
-         [else (error 'insert-casts/Mbox-set!T
+
+
+;;        <<<<<<< HEAD
```

Comment:
`git checkout -b <some new-branch-name> <hash of the commit you want to update>`
`git reset --soft HEAD^`
do the work and `git add` it
`git commit -m "<some message>"`
`git checkout <the PR branch>`
`git rebase -i <new-branch-name>` and delete the line that refers to the original problematic commit
that should be it, but if there is some conflict, fix and stage it and git rebase --continue until it succeeds.
Finally git push -f